index.html

```diff
@@ -121,7 +121,7 @@
         authorizing as necessary across the flow.
       </p>
       <p>
-        The details of how to fulfill a payment request for a given <a>payment
+        The details of how to fulfil a payment request for a given <a>payment
```

Comment:
Ah right, my editor is misconfigured I think: https://en.wiktionary.org/wiki/fulfil

We should change those to en-us. Or is there a difference between "fulfill" and "fulfil" that I don't know about?